src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ElevatorSubsystem extends SubsystemBase{
+    private static ElevatorSubsystem elevatorSubsystem;
+
+    public SkyLimelight s_Limelight = SkyLimelight.getInstance();
+    public Swerve s_Swerve = Swerve.getInstance();
+
+    private CANSparkMax elevator;
+
+    public static ElevatorSubsystem getInstance(){
+        if (elevatorSubsystem == null){
+            elevatorSubsystem = new ElevatorSubsystem();
+        } 
+        return (elevatorSubsystem);
+    }    
+
+    public void elevatorSubsystem(){
```

Comment:
Is this supposed to be the constructor? If so, it should be _public ElevatorSubsystem()_